ClientSide(FrontEnd)/clay_/src/components/forgot.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './forgot.css';
+import imag from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-06 17-07-16.png';
```

Comment:
use proper naming standerd for the import component from one component to another

Reply:
that is a unwanted import i used i removed it